types/gatherer.d.ts

```diff
@@ -41,6 +42,8 @@ declare global {
       gatherMode: GatherMode;
       /** The connection to the page being analyzed. */
       driver: FRTransitionalDriver;
+      computedCache: Map<string, ArbitraryEqualityMap>;
+      settings: Config.Settings;
```

Comment:
after we fix the `LH.Audit.Context` type dependency, this can be removed for a bit :)